xbmc/settings/DisplaySettings.cpp

```diff
@@ -620,6 +620,40 @@ RESOLUTION CDisplaySettings::GetResolutionFromString(const std::string &strResol
   return RES_DESKTOP;
 }
 
+RESOLUTION CDisplaySettings::GetResolutionFromStringIgnoreScreen(const std::string &strResolution)
```

Comment:
let's take the opportunity and remove screen entirely from resolutions. it makes no sense to load resolution from a screen the app is not running on.